### PR TITLE
URL-decode path from API

### PR DESCRIPTION
**Pull request by TobTobXX**
See #4753.

### CODE REVIEW

File: app/user_manager.py [hunk 1]
```diff
@@ -139,6 +140,9 @@ def get_user_data_path(request, check_exists = False, param = "file"):
             if not file:
                 return web.Response(status=400)
                 
+            # Decode URL-encoded file path
+            file = urllib.parse.unquote(file);
```
Comment (renefadmb2bcom) [inline app/user_manager.py:144]: You don't need that semicolon in python. :)
I would remove that.

Comment (renefadmb2bcom) [inline app/user_manager.py:144]: You don't need that semicolon in python. :)
I would remove that.

```suggestion
            file = urllib.parse.unquote(file)
```

**Comment by w1gs** [comment]
Hey any updates on this? It's currently breaking my ComfyUI setup and makes it hard to update without it. If needed I can make the changes and submit another PR if it'll push this along faster.

**Comment by TobTobXX** [comment]
> It's currently breaking my ComfyUI setup and makes it hard to update without it.

Tip: When running into breaking bugs like this, just download the fixing commit as a patch and apply it yourself.
This is a Dockerfile snippet that does this:
```Dockerfile
# Apply patches
COPY *.patch ./
RUN git apply *.patch && rm *.patch
```

File: app/user_manager.py (hunk 1 above)
Comment (mcmonkey4eva) [reply app/user_manager.py:144]: @TobTobXX not sure why you marked this as resolved but didn't actually resolve it? There's still a semicolon there

**Comment by mcmonkey4eva** [comment]
https://github.com/comfyanonymous/ComfyUI/pull/4801 replaced this

**Pull request closed without merging**